exercises/risk-risiko.cpp

```diff
@@ -27,3 +27,107 @@
   M 3 vs 3 => blue win
   O 2 vs 1 => red win
 */
+
+/*
+  Write a program that simulates a risk/risiko fight using 6 dices.
+
+  How does it work?
+  When a player attacks another player he uses 3 dices, the red is always the attacker and the blue is the defender.
+
+  You have to compare the dice with the highest number to simulate the fight.
+  N = first highest number
+  M = second highest number
+  O = third highest number
+
+  If the numbers are equal, the defensor (blue) wins.
+
+  Output:
+  Red dices:
+  6 (N)
+  3 (M)
+  2 (O)
+
+  Blue dices:
+  5 (N)
+  3 (M)
+  1 (O)
+
+    R    B
+  N 6 vs 5 => red win
+  M 3 vs 3 => blue win
+  O 2 vs 1 => red win
+*/
+
+#define EXIT_SUCCESS 0
+#include <iostream>
+#include <cstdlib>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+class Risiko
+{
+private:
+    vector<int> blue;
+    vector<int> red;
+    int attacker_points;
```

Comment:
Per quanto apprezzi l'utilizzo della OOP (Object Oriented Programming), temo che in questo caso sia un po' un'overkill: tutte le proprietà private della classe potrebbero tranquillamente essere spostate dentro il metodo `Attack`.